images/build/debian-base/buster/Dockerfile.build

```diff
@@ -16,7 +16,6 @@ ARG BASEIMAGE
 FROM $BASEIMAGE
 
 ARG ARCH
-COPY qemu-$ARCH-static /usr/bin/
```

Comment:
eventually tracked this back to https://github.com/kubernetes/kubernetes/blame/0f413e5940e18c947bf8e94e9f6c35a574de1baf/build/debian-base/Dockerfile.build#L20 (bit of a pain with the repo move and all),

still not clear why that was done even then. it looks like it was copied from debian-iptables back then https://github.com/kubernetes/kubernetes/pull/41915

Reply:
https://github.com/kubernetes/kubernetes/commit/560268e036f9bbf40e03f39c9386e0a91bfb3938 / https://github.com/kubernetes/kubernetes/pull/21617

looks like this just predates the idea of running multiarch qemu setup seperately from the image build, these never needed to be in any of the component base images, as suspected ...

Reply:
so about this I'm also not sure why this is necessary (or if this is still necessary for cross builds).

I have a RPI3 here, maybe will try later to build the images on amd64 and check if they still work properly on arm64

Reply:
I don't believe this was ever necessary versus doing it in some process external to the image, which we definitely do now.

Reply:
This is safe to remove, I just wanted to understand how it ever got there in case we missed something, because it doesn't make much sense today 🙃 

You can see the history above.

Reply:
@BenTheElder that was required before `buildx` to inject the `qemu-user-static` binary for the platform
https://github.com/kubernetes/kubernetes/blob/0f413e5940e18c947bf8e94e9f6c35a574de1baf/build/debian-base/Makefile#L83-L86

Reply:
@rikatz if you remove the binary, then you can also drop https://github.com/kubernetes/release/blob/ae29abf72e941af1565794fcbc7102ca69804e64/images/build/debian-base/Makefile#L81-L83

Reply:
I don't think it was required even before buildx, we do that by seperately running a qemu setup outside of the base image.